### PR TITLE
Dedicate Riak 2.2.5 to Andy

### PR DESCRIPTION
**Pull request by jplock**
It's not much, but felt appropriate to me considering what he created.

### CODE REVIEW

**Review by ghost — APPROVED** [review]
+1 I approve of this message

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Riak KV 2.2.5 Release Notes
+#### This released as been dedicated to Andy Gross. Thank you and RIP.
```
Comment (ghost) [inline RELEASE-NOTES.md]: #### This released is dedicated to the memory of Andy Gross. Thank you and RIP.

**Comment by russelldb** [comment]
+1, that's a lovely idea. Thank you!

File: RELEASE-NOTES.md (hunk 1 above)
Comment (jrwest) [reply RELEASE-NOTES.md]: "released" -> "release"

**Pull request merged**